File: authorization-server/src/midlewares/cors.py
from aiohttp import web
import aiohttp_cors

from src.config.serverConfig import ALLOWED_ORIGIN_CORS

def add_cors_to(app: web.Application) -> None:
    cors = aiohttp_cors.setup(app, defaults={
        origin: aiohttp_cors.ResourceOptions(
            expose_headers="*",
            allow_headers=["content-type", "authorization"],
            allow_methods=["POST", "GET", "OPTIONS"]
        ) for origin in ALLOWED_ORIGIN_CORS
    })

    for route in list(app.router.routes()):
        cors.add(route)

File: main-server-admin/src/middlewares/CORS.js
import { ALLOWED_ORIGIN_CORS } from "./../config/serverConfig.js";

function CORS(req, res, next) {
	if (ALLOWED_ORIGIN_CORS.includes(req.headers.origin)) {
		res.header('Access-Control-Allow-Origin', ALLOWED_ORIGIN_CORS);
	} else {
		res.header('Access-Control-Allow-Origin', '')
	}
    
    res.header('Access-Control-Allow-Methods', 'GET, POST, OPTIONS');
    res.header('Access-Control-Allow-Headers', 'Authorization, Origin, X-Requested-With, Content-Type, Accept');
    next();
}

export default CORS;